package.json

```diff
@@ -27,44 +27,45 @@
   ],
   "dependencies": {
```

Comment:
If I recall correctly, this change to package.json updates all libraries with more recent versions. To put it the other way, libraries that aren't upgraded such as concat-stream are already the most recent available. I'm noting it to confirm that point because it is best to update libraries when possible to get the benefit of bug fixes in those libraries.